cli/command/formatter/namespace.go

```diff
@@ -80,6 +81,13 @@ func (c *namespaceContext) Labels() string {
 	for k, v := range c.v.Labels {
 		joinLabels = append(joinLabels, fmt.Sprintf("%s=%s", k, v))
 	}
+
+	sort.SliceStable(joinLabels, func(i, j int) bool {
+		fst := strings.Split(joinLabels[i], "=")[0]
+		snd := strings.Split(joinLabels[j], "=")[0]
+		return fst < snd
+	})
+
```

Comment:
Looks like you could use `writeLabels()` here as well?

Reply:
Oops, good catch!